### ISSUE TITLE
Custom user profile fields won't save

### ISSUE THREAD

**Issue by elnino79** [cscart/cscart-boilerplate#76]
After adding custom fields with field name,  user cannot save any data in this field.
IMHO the reason is that Boilerplate use as name "field name" instead "field id".
Eg.: 
Boilerplate: 
![image](https://cloud.githubusercontent.com/assets/4244640/23700812/7576f4ac-03f4-11e7-8a42-fb15b7574220.png)
Responsive:
![image](https://cloud.githubusercontent.com/assets/4244640/23700821/7bd613aa-03f4-11e7-9022-aad7b5e86f86.png)
In my case if replace in file /design/themes/cscart-boilerplate/templates/views/profiles/components/profile_fields.tpl (52-60)
> {if $field.field_name}
>   {assign var="data_name" value="user_data"}
>   {assign var="data_id" value=$field.field_name}
>   {assign var="value" value=$user_data.$data_id}
> {else}
>   {assign var="data_name" value="user_data[fields]"}
>   {assign var="data_id" value=$field.field_id}
>   {assign var="value" value=$user_data.fields.$data_id}
> {/if}

with:

> {assign var="data_name" value="user_data[fields]"}
> {assign var="data_id" value=$field.field_id}
> {assign var="value" value=$user_data.fields.$data_id}

**Comment by elnino79** [comment]
I found in responsive templates solution, that is compatibile with old templates:
```
{if $field.field_name && $field.is_default == 'Y'}
    {assign var="data_name" value="user_data"}
    {assign var="data_id" value=$field.field_name}
    {assign var="value" value=$user_data.$data_id}
{else}
    {assign var="data_name" value="user_data[fields]"}
    {assign var="data_id" value=$field.field_id}
    {assign var="value" value=$user_data.fields.$data_id}
{/if}
```

**Comment by kugimiya** [comment]
Hello, @elnino79 
Thank you for your report. The fix is included in the pull request #94 .
